Source/WebCore/page/LocalFrame.h

```diff
@@ -29,6 +29,7 @@
 
 #include <WebCore/DOMPasteAccess.h>
 #include <WebCore/Frame.h>
+#include <WebCore/ReferrerPolicy.h>
```

Comment:
Can this be forward declared instead of including headers from commonly-included headers?